.github/workflows/dart.yml

```
@@ -16,7 +16,21 @@ env:
   DEV_SEED: ${{ secrets.DEV_SEED }}
 
 jobs:
+  prepare-isolated-resolution:
+    name: Prepare isolated resolution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)